### DISCUSSION TITLE
routing key in binding

### DISCUSSION THREAD

**Discussion by textux** [rabbitmq/messaging-topology-operator#318] [Q&A] [answered]
for binding a queue to an exchange, where do I put the routing key?
source: is the exchange and destination is the queue, but there should be a field for one or more routing keys

**Comment by ChunyiLyu** [comment, marked as answer]
It's `spec.routingKey`. You can check the bindings (or any other crd) schema by `kubectl describe`, e.g.`kubectl describe crd bindings.rabbitmq.com`.
